% Chapter 2

\chapter{Materials and methods} % Main chapter title
\label{Chapter2} % For referencing the chapter elsewhere, use \ref{Chapter2}

\bfseries{Cell line generation}\\
\normalfont Stable cell lines expressing the wildtype and mutant GLUT1 were kindly provided by Katrina Meyer and Markus Landthaler at Max Delbr\"{u}ck Center. In brief, the gene \textit{SLC2A} was purchased from Harvard Plasmid Repository and a stop codon was added with the primers listed in Table~\ref{tab:primers}. The P485L mutation was introduced by polymerase chain reaction-directed mutagenesis with the primers in Table~\ref{tab:primers}.
\begin{table}[h]
\captionsetup{font=normalsize}
\caption{Primer sequences for GLUT1 cloning.}
\label{tab:primers}
\small
\centering
\begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}l}
\toprule
\tabhead{Primer} & \tabhead{Sequence from 5' to 3'}\\
\midrule
Stop codon forward & TCCCAAGTGTAATTGCCAACTTTCTTGTACAAAGTTG\\
Stop codon reverse & ATCAGCCCCCAGGGGATG\\
P485L forward & CTGTTCCATCtCCTGGGGGCT\\
P485L reverse & CTCCTCGGGTGTCTTGTCAC\\
\bottomrule\\
\end{tabular*}
\end{table}
\begin{figure}[h]
\captionsetup{font=normalsize}\centering\includegraphics[scale=0.9]{Figures/vectors}\caption{Graphic maps of the recombinant plasmids.}\label{fig:vectors}\end{figure}
The wildtype and mutant \textit{SLC2A} coding sequences were recombined into the vector pDEST-pcDNA5-BirA-FLAG N-term~\cite{Couzens} using Gateway cloning system (ThermoFisher), as illustrated in Figure~\ref{fig:vectors}. Stable HEK293 cell lines were generated using the Flp-In T-REx system. A control 293T-REx cell line was generated in a similar manner wildtypewith an integrated transgene for the inducible expression of GFP1-10.
\\
\\*
\bfseries{Cell culture}\\
\normalfont HEK293 cells were cultured at \SI{37}{\celsius} and 5\% CO\textsubscript{2} in T-75 flasks (CELLSTAR) at approximately 10\% confluency in DMEM (Life Technologies) complemented with 10\% fetal bovine serum (Pan-Biotech). The medium is referred to complete DMEM in the following. Cells were routinely passaged twice a week as follows: the medium was aspirated and the cells were briefly washed with 4 mL sterile pre-warmed PBS (Life Technologies). To detach the cells 1 mL trypsin-EDTA (0.05\%, Life Technologies) was added and the flask was placed in an incubator at \SI{37}{\celsius} and 5\% CO\textsubscript{2} for 1 min. Trypsin was then inactivated with 9 mL pre-warmed complete DMEM. The medium was gently pipetted to the bottom of the flask in order to recover all the cells and homogenize them. 1 mL of the cell suspension was transferred into 10 mL fresh warm complete DMEM in a new culture flask which was then placed back in the incubator. The cells would reach approximately 80\% confluency before the next passaging.

Cells used for SILAC based experiments were cultured in SILAC DMEM (Life Technologies) complemented with glutamine (Glutamax, Life Technologies), pyruvate (Life Technologies), non-essential amino acids (Life Technologies) and 10\% dialyzed fetal bovine serum (Pan-Biotech). In addition, L-arginine (Arg0, Sigma-Aldrich) and L-lysine (Lys0, Sigma-Aldrich) were added to the Light SILAC DMEM to a final concentration of 0.199 mM and 0.339 mM, respectively. Alternatively, Arg6 and Lys4 or Arg10 and Lys8 were added in place of their Light counterparts to make Medium-heavy SILAC DMEM and Heave SILAC DMEM, respectively. PBS-EDTA (Lonza) was used to detach the cells when passaging as trypsin may contain amino acids. After six passages cells were fully labeled as assessed by MS.
% From Koshi: stock conc. are Lys 0.798M, Arg 0.398M (my dilution is 1:2000). DMEM conc: Lys 146mg/L, Arg 84mg/L; if 1:3000 dilution, final conc. should be 1/4 of standard DMEM conc.

For the doxycycline induction experiments unlabeled HEK cells were cultured in 6-well plates. The cells were grown to approximately 50\% confluency on the second day after seeding. The medium was removed and 2 mL complete DMEM containing doxycycline (Sigma-Aldrich) was carefully added to each well. After 24 hr or 48 hr the cells were harvested for western blot analysis.

For the BioID experiments fully labeled HEK293 cells were seeded in 15 cm plates with approximately 25\% density. Two plates were used for each condition. HEK293 stable cells expressing GFP1-10 were labeled in Light SILAC DMEM and used as negative control cells. Besides the Light control cells, in the forward experiment mutant GLUT1 cells were cultured in Medium-heavy SILAC DMEM and mutant GLUT1 cells were cultured in Heavy SILAC DMEM, while in the reverse label-swap experiment mutant GLUT1 cells were cultured in Medium-heavy SILAC DMEM and mutant GLUT1 cells were cultured in Heavy SILAC DMEM. The cells were grown to 40\%-50\% confluency before being treated with 0.1 {}\textmu g/mL doxycycline and 1 mM biotin (ThermoFisher). After 24 hr, the cells were scraped in ice-cold PBS and spun down at 300 rcf, \SI{4}{\celsius} for 4 min. After resuspension in ice-cold PBS, the triple SILAC labels were combined into a forward and a reverse experiment. The cells were spun down again and the cell pellets were collected for further BioID purification and MS analysis.

For the immunofluorescence experiments sterile glass coverslips (Roth, 18 mm diameter, 0.170 mm thickness) were placed in a 12-well plate. Poly-L-lysine (0.01\%, Sigma-Aldrich) was added to each well to cover the coverslips. After incubation at room temperature for 5 min, poly-L-lysine was recovered from the wells and stored at \SI{4}{\celsius}. The coverslips were washed twice with sterile H\textsubscript{2}O before being air-dried completely. HEK293 cells were then counted and seeded in the plate with approximately 25\% density. The cells were treated with 0.1 {}\textmu g/mL doxycycline on the second day and subjected to subsequent immunostaining on the third day.
\\
\\*
\bfseries{Western blot}\\
\normalfont Cells were grown in 6-well plates as described above. After doxycycline or inhibitor treatment, cells were scraped in ice-cold PBS and spun down at 300 rcf, \SI{4}{\celsius} for 4 min. Cell pellets were lyzed for 15 min at room temperature in lysis buffer (50 mM ABC solution, 2\% SDS, supplemented with 0.5 {}\textmu L benzonase (Sigma-Aldrich) and 0.2 tablet protease inhibitors (Roche) per 2 mL lysis buffer). Lysates were spun down at 140 000 rpm for 15 min to remove cell debris and supernatants were transferred to new Eppendorf tubes. For each SDS-PAGE sample, 15 {}\textmu L supernatant was diluted in LDS sample buffer (NOVEX) complemented with 1 {}\textmu L 1M DTT (Sigma-Aldrich) before being heated at \SI{70}{\celsius} for 10 min in a thermoblock. Samples were then loaded onto a 4\%-12\% gel (NOVEX) along with protein ladders (PageRuler Plus, ThermoFisher). Proteins were separated using electrophoresis for 90 min at 150 V in 400 mL MES running buffer (ThermoFisher). 

Before transferring a PVDF membrane (Merck Millipore) was activated in methanol for 1 min and equilibrated in transfer buffer (25 mM Tris-HCl, 190 mM glycine, 20\% methanol, pH 8.3) for 10 min. Whatman filter papers and sponges were also soaked in transfer buffer for 10 min. A tank blotting system (Invitrogen) was used to transfer the separated proteins to the membrane. In short, a transfer sandwich was prepared with the membrane on the cathode and the gel on the anode. Air bubbles were remove with a roller between the gel and the membrane. The cassette was then placed in the transfer tank on ice and the proteins were transferred at a constant current of 250 mA for 2 hr.

After transferring the membrane was blocked in 5\% milk powder in TBST (150 mM sodium chloride, 20 mM Tris-HCl, 0.1\% Tween-20, pH 7.6) at room temperature for 30 min. The membrane was then incubated with the primary antibody diluted in blocking buffer while rotating at \SI{4}{\celsius}. The membrane was washed 3 times for 5 min in TBST before being incubated at room temperature for 1 hr with the HRP-conjugated secondary antibody diluted in blocking buffer. The membrane was washed again before the chemiluminescence substrate (PerkinElmer) was applied to the membrane. The chemiluminescent signals were captured using a ChemiDoc MP Imaging System (Bio-Rad) and analyzed with Image Lab software version 5.2.1. The primary and secondary antibodies used in this thesis are summarized in Table~\ref{tab:antibodies}.
\begin{table}[h]
\captionsetup{font=normalsize}
\caption{Antibodies used for western blot and their dilutions.}
\label{tab:antibodies}
\small
\centering
\begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}lll}
\toprule
\tabhead{Antibodies} & \tabhead{Source} & \tabhead{Dilution} & \tabhead{Conjugate}\\
\midrule
Rabbit polyclonal anti-FLAG & Cell Signaling Technology & 1:1000 & HRP\\
Rabbit polyclonal anti-LC3 & Novus Biologicals & 1:1000 & \\
Mouse monoclonal anti-\textbeta-actin & Sigma-Aldrich & 1:10 000 & \\
Anti-rabbit IgG & GE Healthcare & 1:10 000 & HRP\\
Anti-mouse IgG & GE Healthcare & 1:25 000 & HRP\\
\bottomrule\\
\end{tabular*}
\end{table}
\\*
\bfseries{BioID and MS analysis}\\
\normalfont Cell pellets from 15 cm plates were incubated at \SI{4}{\celsius} in 1.5 mL of RIPA buffer (50 mM tris-HCl, 150 mM NaCl, 1\% NP-40, 1 mM EDTA, 1 mM EGTA, 0.1\% SDS, 1\% sodium deoxycholate, pH 7.5, per 10 mL supplemented with 1 tablet of protease inhibitors and 3 {}\textmu L benzonase). After incubation with agitation for 1 hr, the lysates were sonicated on ice for 3 min. The lysates were then centrifuged for 20 min at 16 100 rcf at \SI{4}{\celsius} and the supernatants were transferred to 2 mL Eppendorf tubes.

For affinity purification a 180 {}\textmu L bed volume of streptavidin T1 magnetic beads (Invitrogen) was washed in PBS twice and RIPA buffer once before being resuspended in 200 {}\textmu L RIPA buffer. 90 {}\textmu L of the beads was added to the forward and reverse samples, respectively. Affinity purification was performed at \SI{4}{\celsius} for 3 hr on a rotating wheel.

The beads were carefully washed twice in RIPA buffer and twice in TAP lysis buffer (50 mM HEPES-KOH, 100 mM KCl, 10\% glycerol, 2 mM EDTA, 0.1\% NP-40, pH 8.0), followed by washing three times in 50 mM ABC to remove all detergents. The beads were resuspended in 200 {}\textmu L ABC and 10 {}\textmu L 10 mM DTT (Sigma-Aldrich) in 50 mM ABC was added to reduce disulfide bonds of proteins. After 30 min incubation in a thermomixer at 30 \SI{30}{\celsius} at 500 rpm, 10 {}\textmu L 55 mM IAA (Sigma-Aldrich) in 50 mM ABC was added to alkylate the cysteine residues and the samples were incubated in the dark at 30 \SI{30}{\celsius} at 500 rpm for 30 min. The samples then were digested with 1.5 {}\textmu g trypsin (Promega) and incubated overnight at 30 \SI{30}{\celsius} at 1100 rpm. On the next day, the tryptic peptides were separated from beads with a magnetic rack and transferred to fresh 1.5 mL tubes. The digestion was stoped by adding 10 {}\textmu L 10\% TFA and loaded on C18 StageTip columns for purification. After washing with sample buffer (3\% TFA, 5\% acetonitrile in H\textsubscript{2}O) twice, the peptides were eluted from the columns into an autosampler plate with 50 {}\textmu L buffer B (0,1\% formic acid, 80\% acetonitrile in H\textsubscript{2}O). A vacuum centrifuge (Eppendorf) was used to evaporate the solvent and 8 {}\textmu L buffer A (5\% acetonitrile, 0,1\% formic acid in H\textsubscript{2}O) was added to the samples.

Peptides were separated on a reverse-phase column on a High Performance Liquid Chromatography (HPLC) system (ThermoScientific) with a gradient set up as the following ratios of buffer B in buffer A: 2 min at 250 {}\textmu L/min with a linear gradient from 5\% to 6\% buffer B, 18 min at 200 {}\textmu L/min with a linear gradient from 6\% to 8\%, 80 min at 200 {}\textmu L/min with a linear gradient from 8\% to 20\%, 80 min at 200 {}\textmu L/min from 20\% to 33\%, 20 min at 200 {}\textmu L/min from 33\% to 45\%, 2 min at 200 {}\textmu L/min from 45\% to 60\%, 1 min at 250 {}\textmu L/min from 60\% to 95\%, 5 min at 250 {}\textmu L/min with 95\% buffer B, 1 min at 250 {}\textmu L/min from 95\% to 75\%, and 5 min at 250 {}\textmu L/min at 75\% buffer B. Peptides were ionized using an electrospray ionization source (ThermoScientific) and analyzed on a Q-exactive plus Orbitrap instrument (ThermoScientific). The mass spectrometer was operated with Xcalibur in data-dependent acquisition mode with the following parameters: a full MS scan (resolution: 70 000, scan range: 300 to 1700 m/z, AGC target: 1 000 000 ions, maximum injection time: 120 ms) followed by MS-MS analysis (resolution: 17 500, AGC target: 100 000 ions, maximum injection time: 60 ms) on the top 10 abundant ions.

The resulting raw files were analyzed using MaxQuant 1.5.2.8~\cite{Cox}. The reference human proteome database, consisting of 159 616 entries, was downloaded on February 28th, 2017 from the UniProt Knowledgebase. Lys4, Arg6 and Lys8, Arg10 were added to the labels. Variable modifications were set as N-terminal acetylation and oxidation of methionines. The \textit{in silico} digestion of the reference database was performed with trypsin/P site-specific cleavage. Default global parameters were kept except that "re-quantify" and "match between runs" were turned on. The false discovery rate, assessed by in-parallel searching in a decoy database generated by reversing the reference database, was set at 0.01 at peptide and protein levels. Following analysis was performed using Persius and R 3.3.3.
% cut off
\\
\\*
%\bfseries{Inhibitor treatment}\\
%\\*
\bfseries{Transferrin uptake}\\
\normalfont After 24 hr of doxycycline induction, the cells were starved for 1 hr in serum-free medium supplemented with 20 mM HEPES
% company
. Transferrin conjugated with Alexa Fluor 568 (Life Technologies) was diluted in starvation buffer to a final concentration of 10 {}\textmu g/mL and droplets of 35 {}\textmu L the transferrin solution were pipetted onto a parafilm in a dark humid chamber. The coverslips were then carefully lifted off the bottom of the plate with forceps and placed face-down on droplets. The humid chamber was incubated at \SI{37}{\celsius} and 5\% CO\textsubscript{2} for 10 min. After transferrin uptake, the coverslips were washed 3 times for 5 min with PBS supplemented with 10 mM MgCl\textsubscript{2} and 10 mM CaCl\textsubscript{2}, followed by standard immunostaining procedures described below.
\\
\\*
\bfseries{Immunostaining and confocal microscopy}\\
\normalfont Prior to staining, cell culture medium was aspirated and cells were washed with pre-warmed PBS once. The cells were fixed in 4\% paraformaldehyde for 15 min at room temperature before being washed 3 times for 5 min in PBS. To permeabilize cells and block unspecific binding sites the cells were incubated for 1 hr at room temperature in blocking buffer consisting of 5\% goat serum (Sigma-Aldrich) and 0.3\% Triton X-100 (Sigma-Aldrich) in PBS. The primary antibody was diluted in antibody dilution buffer consisting of 1\% BSA - Fraction V (Sigma-Aldrich) and 0.3\% Triton X-100 in PBS, as indicated in Table~\ref{tab:IF}. The coverslips were placed face-down on droplets of 50 {}\textmu L primary antibody solution on a piece of parafilm in a dark humid chamber. After 1 hr incubation at room temperature, the coverslips were placed face-up in the plate and washed 3 times for 5 min in PBS. Similarly, the coverslips were incubated with secondary antibody solution for 1 hr at room temperature in the humid chamber before being washed for 5 min in PBS. The coverslips were then counterstained with 
% conc. 
DAPI in PBS for 3 min and washed for 10 min in PBS. Finally, the coverslips were washed briefly with MilliQ H\textsubscript{2}O and mounted with ProLong Gold Antifade Mountant (Life Technologies) on slides. After overnight incubation, the slides were stored at \SI{4}{\celsius} in the dark.

\begin{table}[h]
\captionsetup{font=normalsize}
\caption{Antibodies for immunofluorescence and their dilutions.}
\label{tab:IF}
\small
\centering
\begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}lll}
\toprule
\tabhead{Antibodies} & \tabhead{Source} & \tabhead{Dilution} & \tabhead{Conjugate}\\
\midrule
Mouse monoclonal anti-FLAG & Sigma-Aldrich & 1:200 & \\
Rabbit monoclonal anti-EEA1 & Cell Signaling Technology & 1:100 & \\
Rabbit monoclonal anti-Rab9 & Cell Signaling Technology & 1:100 & \\
Rabbit monoclonal anti-Rab11 & Cell Signaling Technology & 1:100 & \\
Rabbit monoclonal anti-LAMP1 & Cell Signaling Technology & 1:100 & \\
Goat anti-Mouse IgG (H+L) & Invitrogen & 1:500 & Alexa Fluor 488\\
Donkey anti-Rabbit IgG (H+L) & Invitrogen & 1:500 & Alexa Fluor 568 \\
\bottomrule
\end{tabular*}
\end{table}
Images in this thesis were acquired using Leica DMI6600 confocal laser scanning microscope with a HCX PL APO 63.0$\times$/1.40 oil objective. As summarized in Table~\ref{tab:laser}, fluorophores were excited using 405 nm laser diode, Argon 488 nm laser (20\% power) and DPSS 561 nm laser and detected using photomultiplier tubes (PMT). Pinhole diameter was set to 95.6 {}\textmu m, scanning mode was unidirectional, line average was 2, sampling speed was 400 Hz, and zoom was 5. The voxel size was set to 48.1 nm (width) $\times$ 48.1 nm (height) $\times$ 125.9 nm (depth).

\begin{table}[h]
\captionsetup{font=normalsize}
\caption{Settings for fluorescence excitation and detection.}
\label{tab:laser}
\small
\centering
\begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}lllll}
\toprule
\tabhead{Fluorophore} & \tabhead{Laser line} & \tabhead{Laser intensity} & \tabhead{PMT} & \tabhead{PMT gain} & \tabhead{PMT offset}\\
\midrule
DAPI & 405 nm & 8.00\% & 413-477 nm & 619 V & -1\\
Alexa 488 & 488 nm & 12.00\% & 506-598 nm & 646 V & -1\\
Alexa 568 & 561 nm & 15.00\% & 580-710 nm & 619 V & -1\\
\bottomrule
\end{tabular*}
\end{table}
The z-stack confocal microscopy images were further analyzed using ImageJ 1.51j. The stack viewing and color options were set as the default configuration of the Bio-Formats plugin. One z slice was selected in each image to represent all stacks. The brightness and contrast of the channels were uniformly adjusted in each staining experiment.
%----------------------------------------------------------------------------------------

% Define some commands to keep the formatting separated from the content 
